osu.Game/OsuGameBase.cs

```diff
@@ -211,21 +211,14 @@ protected override void LoadComplete()
 
             GlobalKeyBindingInputManager globalBinding;
 
-            base.Content.Add(new DrawSizePreservingFillContainer
+            CursorVisualiser = new OsuCursorVisualiser { RelativeSizeAxes = Axes.Both };
+            CursorVisualiser.Child = globalBinding = new GlobalKeyBindingInputManager(this)
             {
-                Children = new Drawable[]
-                {
-                    Cursor = new MenuCursor(),
-                    globalBinding = new GlobalKeyBindingInputManager(this)
-                    {
-                        RelativeSizeAxes = Axes.Both,
-                        Child = content = new OsuTooltipContainer(Cursor)
-                        {
-                            RelativeSizeAxes = Axes.Both,
-                        }
-                    }
-                }
-            });
+                RelativeSizeAxes = Axes.Both,
+                Child = content = new OsuTooltipContainer(CursorVisualiser.Cursor) { RelativeSizeAxes = Axes.Both　}
+            };
+
+            base.Content.Add(new DrawSizePreservingFillContainer { Child = CursorVisualiser });

```

Comment:
can this be combined with the previous statement?

```base.Contnt = new DSPFC { Child = CursorVisualiser = new ...```

Reply:
It wouldn't help too much, the issue is that the tooltip container requires the cursor. So at most you'd be able to combine the instantiation of the cursor container which leads to a 2 line increase...